### PR TITLE
f-checkout@v0.172.0 - Update to new version of Mega Modal that outputs min js

### PR DESCRIPTION
**Pull request by matthewhardern**
This PR is a test to see if the reduced bundle size of f-checkout improves memory usage and stops builds failing

### CODE REVIEW

**Review by JoshuaNg2332 — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

File: packages/components/molecules/f-mega-modal/vue.config.js
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/vue.config.js:8]: Are you able to explain the intention of this change? Is it doing something specifically to help with bundle sizes?

Comment (siggerzz) [reply packages/components/molecules/f-mega-modal/vue.config.js:8]: Correct. It splits the anything within `node_modules` into its own bundle.

Currently CircleCI has issues building f-checkout / f-searchbox due to memory issues on the free plan, so have to focus on bundle optimisation to unblock CI until we get approval for a paid plan 👍🏼.


There's defiantly another conversation to be had on bundle optimisation I think. From mine and @matthewhardern's initial investigation, there's a lot more we could be doing around splitting / compression.

**Pull request closed without merging**
